scripts/akv/down.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+akv-down() {
+    set -e
+
+    REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../..")"
+
+    source $REPO_ROOT/services/cacitesting.env
+    AKV_NAME=${AKV_NAME:-$1}
```

Comment:
AKV_VAULT_NAME for clarity.

Reply:
https://github.com/microsoft/azure-privacy-sandbox-kms/pull/233/commits/fa3f24570c49c3fb9538db1c25a5f5aa39658ae0